src/helpers/formatters/file.js

```diff
@@ -1,17 +1,17 @@
 import fs from 'fs'
+import * as musicMetadata from 'music-metadata'
 
-export const tags = (tags, file) => {
-  const image = formatImage(
-    tags.APIC
-  )
+export function mapMetadataTags (tags, file) {
+  const coverPicture = musicMetadata.selectCover(tags.picture)
+  const image = coverPicture ? `data:${coverPicture.format};base64,${coverPicture.data.toString('base64')}` : null
```

Comment:
@Borewit Please bring back `formatImage` function and place thaht logic in here.

Reply:
Will do